src/olympia/editors/tests/test_helpers.py

```diff
@@ -137,14 +137,30 @@ def test_last_review(self):
         doc = pq(self.table.render_last_review(row))
         assert doc.text() == u'0.34.3b on 2016-01-01'
 
-    def test_authors(self):
+    def test_authors_few(self):
         row = Mock()
-        row.authors = [(123, 'bob'), (345, 'steve')]
+        row.authors = [(123, 'bob'), (456, 'steve')]
         doc = pq(self.table.render_authors(row))
-        url = UserProfile.create_user_url(123, username='bob')
-        assert doc('span').text() == 'bob ...'
-        assert doc('span a').attr('href') == url
-        assert doc('span').attr('title') == 'bob steve '
+        assert doc('span').text() == 'bob steve'
+        assert doc('span a:eq(0)').attr('href') == UserProfile.create_user_url(
+            123, username='bob')
+        assert doc('span a:eq(1)').attr('href') == UserProfile.create_user_url(
+            456, username='steve')
+        assert doc('span').attr('title') == 'bob steve'
+
+    def test_authors_four(self):
```

Comment:
There _should_ but I'm still not certain how they came about - I guessed it was due to versions with multiple files that I was (unnecessarily) `JOIN`ing so I split out the sql construction (in another pull) but the bug persisted so I gave up trying to understand the root cause and just fixed the symptom.  The test would need to be in test_models if it existed - in `.authors` they're already de-duped.